frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -128,3 +121,21 @@ export const InventoryItemLoading: React.FC = require('@console/shared/src/compo
   .InventoryItemLoading;
 
 export { useFlag } from '../utils/flags';
+
+/**
+ * A lazy component that creates a monaco editor
+ * @example
+ * ```tsx
+ * <ResourceYAMLEditor
+ *     content={yamlString}
+ *     header="Create resource"
+ *     onSave={(content) => updateResource(content)}
+ *    />
+ * ```
+ * @param {EditYAMLProps['content']} content - Stringified YAML/Object to be shown by the Editor
```

Comment:
I'd suggest moving these JSDocs to the relevant Props properties, see my comment below.